discord.go

```diff
@@ -2,12 +2,14 @@ package servermanager
 
 import (
 	"errors"
-	"net/url"
-	"time"
-
+	"fmt"
 	"github.com/Clinet/discordgo-embed"
 	"github.com/bwmarrin/discordgo"
+	"github.com/google/uuid"
 	"github.com/sirupsen/logrus"
+	"net/url"
```

Comment:
can you group the stdlib imports at the top? i know i'm being picky haha.

i tend to do:

* stdlib imports at the top
* a gap, then sub-package imports
* another gap (if necessary) and then third party imports

so like
```go
import (
    "errors"
    "url"

    "github.com/cj123/assetto-server-manager/pkg/udp"

    "github.com/google/uuid"
    "github.com/sirupsen/logrus"
)
```

Reply:
Okie dokie